qiskit/pulse/commands/frame_change.py

```diff
@@ -58,7 +58,9 @@ def __hash__(self):
         return hash((super().__hash__(), self.phase))
 
     def __repr__(self):
-        return '%s(%s, phase=%.3f)' % (self.__class__.__name__, self.name, self.phase)
+        return '%s(phase=%.3f, name="%s")' % (self.__class__.__name__,
```

Comment:
can we add the unit of phase, ie `rad.`?

Reply:
No, violates the python data model.